.github/workflows/ci.yml

```diff
@@ -289,6 +289,9 @@ jobs:
             env:
               DEPLOY_TOOLSTATES_JSON: toolstates-linux.json
             os: ubuntu-20.04-xl
+          - name: x86_64-uefi
+            os: ubuntu-20.04-xl
+            env: {}
```

Comment:
This might fit into the `test-various` job? That currently 37minutes so it's one of the shorter-running jobs.

Reply:
@nicholasbishop can you please make this change?

Reply:
Done.

One thing to note is that the `test-various` Dockerfile is using Ubuntu 20.04, which provides a quite old QEMU (4.2.1). That version of QEMU likes to crash on exit with OMVF, we ran into that problem in the CI for uefi-rs in the past. I changed the test to not check the exit status of the VM, which I think is fine since we're checking the stdout for a specific string anyway.